gui/src-tauri/src/main.rs

```diff
@@ -33,16 +34,26 @@ static POSSIBLE_TITLES: &[&str] = &[
 	"never gonna let you down",
 	"uwu sowwy",
 ];
+shadow!(build);
+// Tauri has a way to return the package.json version, but it's not a constant...
+const VERSION: &str = if build::TAG.is_empty() {
+	formatcp!("git-{}", build::SHORT_COMMIT)
```

Comment:
can't you just use `concat!()` here? Avoids the need for another dependency

Reply:
Nah concat doesnt work aside from literals